client/components/domains/search-filters/tld-filter-bar.jsx

```diff
@@ -99,10 +101,23 @@ export class TldFilterBar extends Component {
 			return this.renderPlaceholder();
 		}
 
-		const className = classNames( 'search-filters__buttons', {
+		const className = classNames( '', {
```

Comment:
The first arg, `'',` doesn't have to be there, right?

Reply:
Thanks, implemented in [517527c](https://github.com/Automattic/wp-calypso/pull/39346/commits/517527c44eb4b7dd43fbf995cfa270113dc23db1).